app/javascript/packs/document-capture-welcome.ts

```diff
@@ -0,0 +1,79 @@
+import { trackEvent } from '@18f/identity-analytics';
+import { hasCamera, isCameraCapableMobile } from '@18f/identity-device';
+
+const GRACE_TIME_FOR_CAMERA_CHECK_MS = 2000;
+const DEVICE_CHECK_EVENT = 'Idv: Mobile device and camera check';
```

Comment:
Couple thoughts:

* We're not 100% consistent, but I'd expect "IdV" as the more conventional capitalization here
* We should consider adding the event to `AnalyticsEvents` and wiring it up in [FrontendLogController::EVENT_MAP](https://github.com/18F/identity-idp/blob/48a8ff55389a93203654ab5787b6750889d0ea25/app/controllers/frontend_log_controller.rb#L8-L23) so that the event is documented and to avoid a "Frontend:" prefix that's prepended to any non-allowlisted events